gym_roboy/envs/msj_env.py

```diff
@@ -69,8 +71,10 @@ def _set_new_goal(self, goal_joint_angle=None):
         if goal_joint_angle is not None:
             self._goal_joint_angle = goal_joint_angle
             return
-        new_joint_angle = np.random.random(self._ros_proxy.DIM_JOINT_ANGLE)
-        self._goal_joint_angle = np.clip(new_joint_angle, -self._max_joint_angle, self._max_joint_angle)
+        new_joint_angle = self._ros_proxy.set_new_goal()
+        self._goal_joint_angle = new_joint_angle
+        self._ros_proxy.forward_new_goal(self._goal_joint_angle)
```

Comment:
Do we still forward a new goal from Python to C++ or rather call a service?

Reply:
Now, I understood, `forward_new_goal` is for rviz. Do we still need to call it on every step (every time we call `_make_obs()` ?? Or is it enough to call it once after we fetch a new goal?

Reply:
One time is enough.